packages/react-native-gesture-handler/apple/RNGestureHandler.h

```diff
@@ -39,8 +39,9 @@
 @protocol RNGestureHandlerEventEmitter
 
 - (void)sendEvent:(nonnull RNGestureHandlerStateChange *)event
-    withActionType:(RNGestureHandlerActionType)actionType
-     forRecognizer:(UIGestureRecognizer *)recognizer;
+       withActionType:(RNGestureHandlerActionType)actionType
+    withAnimatedEvent:(BOOL)useAnimatedEvent
```

Comment:
```suggestion
    forAnimated:(BOOL)forAnimated
```

Reply:
Ok, looks better. Done in [118467e](https://github.com/software-mansion/react-native-gesture-handler/pull/3646/commits/118467e3448bf7d427a3a06737fd0d4ffd62afd6)